packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -459,14 +481,41 @@ function UserMenu({
             {isGovernment ? 'Cloud for Government' : 'Cloud'}
           </MenuItem>
         )}
+
+        <MenuItem
+          {...subMenuContainer.prop}
+          {...sharedProps}
+          size="large"
+          active={isDocs}
+          disabled={!account}
+          href="https://docs.mongodb.com"
```

Comment:
now could be a good time to factor out these urls into constants

Reply:
👍